terragrunt/modules/ecs-app/ci.tf

```diff
@@ -0,0 +1,44 @@
+// ECR repository which will store the Docker image powering the application.
+
+module "ecr" {
+  source = "../ecr-repo"
+  name   = var.name
+}
+
+// IAM User used by GitHub Actions to pull and push images to ECR, and to
+// restart the ECS service once the image is uploaded. The credentials will
+// be added automatically to the GitHub Actions secrets.
+
+module "iam_ci" {
+  source     = "../gha-iam-user"
+  org        = split("/", var.repo)[0]
+  repo       = split("/", var.repo)[1]
+  env_prefix = var.env == "dev" ? "DEV" : null
```

Comment:
Same question here as above regarding the naming of the environment: `dev` vs. `staging.`